src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, onTabSelected, selectedTabId }) => {
+  const currentTab = tabs.find(tab => tab.id === selectedTabId)
+  || tabs[0];
+
+  return (
+    <div className="section">
+      <h1 className="title">
+        Selected tab is
```

Comment:
It is better to avoid constructions like this, better use {`Selected tab is ${currentTab.title}`}